QBTracker/QubitSDK/APIClient/Entities/QBEventEntity.swift

```diff
@@ -17,7 +17,7 @@ enum QBEventType: String {
         switch type {
         case "qubit.session":
             self = .session
-        case "View":
+        case _ where type.hasSuffix("View"):
```

Comment:
Btw across the rest of the app the way we identify a view event is using the regex `/^([^.]+\.)?[a-z]{2}View$/`

Reply:
@alanclarke Do you want me to change it to reflect those kind of behavior?

Reply:
This would be more robust and more cohesive with the rest of our stack, so I think it is a worthwhile change, as long as it is a quick change.

Reply:
yes please @mbalawajder

Reply:
@mbalawajder i understand there were some questions around this, and whether it would be a mistake to narrow the definition in this way. I can confirm that this regex is used across the app to detect a valid view event. The definition of a view event is determined by whichever events are defined here: https://github.com/qubitdigital/qprotocol-schemas and as you can see it is by design that all view events strictly follow the pattern described by the regex. Anything else will not be valid and will be caught by our gatekeeper event validation service

Reply:
@alanclarke I don't have an access to the file you mentioned, but just checked out against Android, and yes, it looks like it's a bug then - that has been wrong since the beginning. Will correct that.